swagger-ui/src/main/resources/index.html

```diff
@@ -1,142 +0,0 @@
-<!--
```

Comment:
@jimmidyson Gotcha. @KurtStam I think it could be worth it to keep the Swagger UI, but Jimmi is right in that it should be a separate GH issue and certainly shouldn't block this PR. The Chrome extension can be a temporary thing and we can look into providing our own later on since it's a lower priority for the MVP. I say get your merge on @jimmidyson .